### PR TITLE
Fix bad indentation on paste

### PR DESCRIPTION
**Pull request by aeisenberg**
I don't fully understand why this is working differently, but these changes enable proper behavior on pasting ql into the editor.

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/master/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [x] Issues have been created for any UI or other user-facing changes made by this pull request.
- [x] `@github/product-docs-dsp` has been cc'd in all issues for UI or other user-facing changes made by this pull request.

### CODE REVIEW

**Comment by aeisenberg** [comment]
Fixes #428

File: extensions/ql-vscode/src/languageSupport.ts [hunk 1]
```diff
@@ -17,11 +17,14 @@ export function install() {
   // setLanguageConfiguration requires a regexp for the wordpattern, not a string
   langConfig.wordPattern = new RegExp(langConfig.wordPattern);
   langConfig.onEnterRules = onEnterRules;
+  langConfig.indentationRules = {
+		increaseIndentPattern: /^((?!.*?\/\*).*\*\/)?\s*[\}\]].*$/,
+		decreaseIndentPattern: /^((?!\/\/).)*(\{[^}"'`]*|\([^)"'`]*|\[[^\]"'`]*)$/
+	};
```
Comment (alexet) [inline extensions/ql-vscode/src/languageSupport.ts:23]: This still looks to me like these are back to front. You seem to have `}` and `]` increasing the indentation and `{` and [`  increasing it.

Comment (jcreedcmu) [reply extensions/ql-vscode/src/languageSupport.ts:23]: These are the same patterns suggested by https://code.visualstudio.com/api/language-extensions/language-configuration-guide so as long as they are swapped as suggested this lgtm.

**Comment by aeisenberg** [comment]
It does make sense to swap them, but that begs the question _why_ things seem to work even when they are mixed up.

**Comment by jcreedcmu** [comment]
For me, they don't; I get incorrect indentation when pasting with them swapped, e.g.
```
predicate foo() {
exists(int x | x = x
)
}
```

**Comment by aeisenberg** [comment]
This is still not perfect, and I think fundamentally there are limitations in how vscode works (that are not present in Eclipse), so we are never going to be able to get a pasting and formatting experience in vscode.

**Comment by jcreedcmu** [comment]
Hm, as of this commit now I can reproduce the bad behavior
```
predicate foo() {
    exists(int x | x = x
    )
    }
```
again, which was what I saw earlier.

**Comment by jcreedcmu** [comment]
Ok --- I think I realize that the failure I'm seeing is (now) simply vscode *not* formatting pasted code in a way that it *could* format it, rather than actively *mis*formatting it. In that case this lgtm.

**Pull request merged**